src/app/exhibitor/_components/ExhibitorTimeline.tsx

```diff
@@ -1,396 +1,226 @@
-							students who make Armada possible in a more informal setting.
-							Welcome!
-						</P>
-					</CollapsibleContent>
-				</Collapsible>
-			</li>
-
-			<li className="mb-2 ms-4">
-				<div className="absolute -start-1.5 mt-1.5 h-3 w-3 rounded-full border border-white bg-melon-700"></div>
-
-				<Collapsible>
-					<CollapsibleTrigger className="rounded px-2 pb-4 text-left">
-						<P className="	text-stone-400">{formatDate(dates.fair.days[1])}</P>
-						<div className="flex w-full justify-between">
-							<h3 className="text-2xl md:text-3xl">Armada fair ends</h3>
-							<div className="flex h-8 w-8 items-center justify-center rounded-full"></div>
-						</div>
-					</CollapsibleTrigger>
-				</Collapsible>
-			</li>
-		</ol>
+		<Accordion
+			type="single"
+			collapsible={true}
+			className="relative mt-10 border-s border-melon-700">
+			<TimelineItem
+				dateString={`Before ${formatDate(dates.ir.start)}`}
+				title="Armada is setting up">
+				<P className="mt-3 text-stone-400">
+					Before the Initial Registration can open, we need to make
+					preparations. We are right now choosing a new project group - 20
+					something students who will work hard all year to make Armada happen.
+				</P>
+				<P className="mt-3 text-stone-400">
+					We will open Initial Registration where you apply to be an exhibitor
+					soon. You can express your interest here, and we will contact you as
+					soon as registration opens!
+				</P>
+			</TimelineItem>
+
+			<TimelineItem
+				dateString={formatDate(dates.ir.start)}
+				title="Initial Registration starts">
+				<P className="mt-3 text-stone-400">
+					Initial Registration is where you apply to be an exhibitor. When you
+					register you commit to be a part of Armada and if given a spot you are
+					expected to exhibit, so wait with registration until you are sure. If
+					you have any questions, do not hesitate to contact{" "}
+					<a
+						className="text-white underline hover:no-underline"
+						href="mailto:sales@armada.nu">
+						sales@armada.nu
+					</a>
+					.
+				</P>
+				<P className="mt-3 text-stone-400">
+					Sadly, we can&apos;t guarantee a spot for everyone that applies. We
+					are right now investigating how many exhibitors we can fit and how big
+					the interest is. We try our best to get a good mix of great exhibitors
+					that make Armada the best place for students to find their dream
+					employer!
+				</P>
+				<P className="mt-3 text-stone-400">
+					During the Initial Registration you don&apos;t need to choose a
+					package, and the packages are outlined{" "}
+					<Link
+						className="text-white underline hover:no-underline"
+						href="/exhibitor/packages">
+						here
+					</Link>{" "}
+					to give you an overview. Prices are set, and small changes can occur
+					in the larger packages.
+				</P>
+				<div className="my-4">
+					<a href="https://register.armada.nu/register">
```

Comment:
nextjs has a builtin `<Link>` component which provides some optimizations, I'm aware that this was here already but it would be great if it could be fixed in this PR

Reply:
https://nextjs.org/docs/app/api-reference/components/link